README.md

```diff
@@ -1 +1,171 @@
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 Math Magician <a name="about-project"></a>
+
+> In this project I will develop a user-friendly single-page application for performing various math calculations, but for this task I setup all the tool needed to start building my project
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+- REACT
+
+### Tech Stack <a name="tech-stack"></a>
+
+> 
+
+<details>
+  <summary>Cient</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/JavaScript">Javascript</a></li>
+    <li><a href="https://react.dev/">React</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Sngle Page Website**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need the following tools:
+- Node.js on your computer
+- Code editor e.g.: VScode
+
+### Setup
+
+Clone this repository to your desired folder by running the following commands in your terminal:
+
+```sh
+  cd your-prefered-folder-name
+  
+  git clone https://github.com/mershark/math-magician.git
+```
+
+### Install
+
+- Install this project with:
+
+```sh
+  cd math-magician
+
+  npm install or npm i
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm run test
+```
+
+### Deployment
+
+You can deploy this project by transferring the files in your local repository onto your server
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- VIDEO WALKTHROUGH -->
+
+## 🚀 Video Walkthrough <a name="live-demo"></a>
+
+([coming soon]())
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo Website <a name="live-demo"></a>
+
+([coming soon]())
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mershark Takyi**
+
+- GitHub: [@githubhandle](https://github.com/mershark)
+- LinkedIn: [Profile link](https://www.linkedin.com/in/mershark/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+- [ ] **Calculator Papge**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] When I try to open `Issue page` then the link is broken please use the correct URL here `https://github.com/mershark/math-magician/issues` as it makes you README file more professional.